packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -71,8 +90,8 @@ export function createTokenIntrospectionMiddleware({
         ) {
           return true
         }
-        return access.actions.find((tokenAction) => {
-          if (tokenAction === requestAction) {
+        return access.actions.find((tokenAction: AccessAction) => {
+          if (tokenAction === requestAction && isActiveTokenInfo(tokenInfo)) {
```

Comment:
```suggestion
          if (tokenAction === requestAction) {
```

Reply:
That does not work because it's within `.find`. Typescript again complains that `tokenInfo.clientId` does not exist on `{active: false}`